android/src/main/java/com/swmansion/rnscreens/SearchBarManager.kt

```diff
@@ -71,45 +84,33 @@ class SearchBarManager : ViewGroupManager<SearchBarView>() {
     }
 
     @ReactProp(name = "placeholder")
-    fun setPlaceholder(view: SearchBarView, placeholder: String?) {
+    override fun setPlaceholder(view: SearchBarView, placeholder: String?) {
         if (placeholder != null) {
             view.placeholder = placeholder
         }
     }
 
     @ReactProp(name = "textColor", customType = "Color")
-    fun setTextColor(view: SearchBarView, color: Int?) {
+    override fun setTextColor(view: SearchBarView, color: Int?) {
         view.textColor = color
     }
 
     @ReactProp(name = "headerIconColor", customType = "Color")
-    fun setHeaderIconColor(view: SearchBarView, color: Int?) {
+    override fun setHeaderIconColor(view: SearchBarView, color: Int?) {
         view.headerIconColor = color
     }
 
     @ReactProp(name = "hintTextColor", customType = "Color")
-    fun setHintTextColor(view: SearchBarView, color: Int?) {
+    override fun setHintTextColor(view: SearchBarView, color: Int?) {
         view.hintTextColor = color
     }
 
     @ReactProp(name = "shouldShowHintSearchIcon")
-    fun setShouldShowHintSearchIcon(view: SearchBarView, shouldShowHintSearchIcon: Boolean?) {
-        view.shouldShowHintSearchIcon = shouldShowHintSearchIcon ?: true
+    override fun setShouldShowHintSearchIcon(view: SearchBarView, value: Boolean) {
+        view.shouldShowHintSearchIcon = value ?: true
     }
 
-    fun setPlacement(view: SearchBarView, placeholder: String?) = Unit
-
-    override fun receiveCommand(root: SearchBarView, commandId: String?, args: ReadableArray?) {
-        when (commandId) {
-            "focus" -> root.handleFocusJsRequest()
-            "blur" -> root.handleBlurJsRequest()
-            "clearText" -> root.handleClearTextJsRequest()
-            "toggleCancelButton" -> root.handleToggleCancelButtonJsRequest(false) // just a dummy argument
-            "setText" -> root.handleSetTextJsRequest(args?.getString(0))
-            "cancelSearch" -> root.handleCancelSearchJsRequest()
-            else -> throw JSApplicationIllegalArgumentException("Unsupported native command received: $commandId")
-        }
-    }
+    override fun setPlacement(view: SearchBarView, placeholder: String?) = Unit
```

Comment:
It seems this one is also iOS only 
```suggestion
    override fun setPlacement(view: SearchBarView, placeholder: String?) {
        logNotAvailable("placement")
    }
```

Idk whats better, truncating implementation down to noop or logging Android error (JS user won't see it anyway). We can discuss it shortly on weekly.

Also I'm not sure whether we can even trigger this function to be called when setting this prop from JS as it is not annotated with `@ReactProp` 😅 TODO: Test whether setting the non-supported prop in JS would trigger this function anyway. If not - we should remove all the "logNotAvailable" implementations (in separate PR).

Reply:
Yeah, actually those are logged 
<img width="1313" alt="image" src="https://github.com/software-mansion/react-native-screens/assets/11800297/5f313dbb-0c64-44d5-b5d3-f894e2d724af">